src/app/app.component.ts

```diff
@@ -1,11 +1,55 @@
-import { Component } from '@angular/core';
-import { Course } from './shared/models/course/course';
+import {
+  Component,
+  OnChanges,
+  OnInit,
+  DoCheck,
+  AfterContentInit,
+  AfterContentChecked,
+  AfterViewInit,
+  AfterViewChecked,
+  OnDestroy
+} from "@angular/core";
+
+import { Course } from "./shared/models/course/course";
 
 @Component({
-  selector: 'app-root',
-  templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss']
+  selector: "app-root",
```

Comment:
As far as I remember the style guide for Angular is recommending the single quotes - ', not double ones - ". Maybe your editor needs to be configured to put single quotes.

Reply:
Done :)